WordPress/Classes/Utility/BuildInformation/FeatureFlagOverrideStore.swift

```diff
@@ -3,12 +3,9 @@ import Foundation
 // Protocol allows easier unit testing, so we can implement mock
 // feature flags to use in test cases.
 //
-protocol OverrideableFlag: CustomStringConvertible {
-    var enabled: Bool { get }
+protocol OverridableFlag: CustomStringConvertible {
+    var originalValue: Bool { get }
```

Comment:
Replacing `enabled` with `originalValue`  is a nice clarity improvement. Thanks!